src/index.js

```diff
@@ -1,14 +1,95 @@
 import './style.css';
 
-const li = document.querySelector('.last');
-const listData = [{ description: 'Wake up at 7:00am', completed: false, index: 0 },
-  { description: 'Work out and shower', completed: false, index: 1 },
-  { description: 'Take breakfast', completed: false, index: 2 },
-  { description: 'Meditation', completed: false, index: 3 },
-];
-function populateData() {
-  listData.forEach((data) => {
-    li.insertAdjacentHTML('beforebegin', `<li><p><input type="checkbox" id="checkbox">${data.description}</p><i class="fa-solid fa-ellipsis-vertical"></i></li>`);
+const addToDo = document.querySelector('.addToDo');
+const ul2 = document.querySelector('.ul-2');
+function getItems() {
+  const items = localStorage.getItem('To Do') ? JSON.parse(localStorage.getItem('To Do')) : [];
+  return items;
+}
+let toDoData = getItems(); // used let as the value would be changed.
+
+function setItems() {
```

Comment:
* It is a [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=Create%20a%20new%20JavaScript%20file%20for%20the%20new%20functionality.) that you Create a new JavaScript file for the new functionality as one of the learning objectives of this milestone is to Use ES6 modules to write modular JavaScript. Kindly create a new file for these functionalities and import them where needed 🙏